src/Publisher.ts

```diff
@@ -395,8 +413,9 @@ export default class Publisher {
 			!inlineMatches &&
 			!dataviewJsMatches &&
 			!inlineJsMatches
-		)
-			return;
+		) {
+			return text;
```

Comment:
This one actually might matter. seems like it drops the whole compilation step if this returns, so type safety actually means something here!!! lmk if this should not return text, and throw error or something